proposals/lip-0063.md

```diff
@@ -537,6 +577,35 @@ def createGenesisDataObj():
     return genesisDataObj
 ```
 
+###### Assets Entry for Interoperability Module
+
+Let `genesisInteroperabilityStoreSchema` be as defined in [LIP 0045](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0045.md#genesis-state-initialization). The `assets` entry for the Interoperability module is added by the logic defined in the function `addInteropModuleEntry` in the following pseudo code. The function `getMainchainID` returns the chain ID of the mainchain in the current network, it is defined in [LIP 0037](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0037.md#mainchain-chain-id).
+
+```python
+def addInteropModuleEntry():
+    InteropObj = {}
+    InteropObj.outboxRootSubstore = []
+    InteropObj.chainDataSubstore = []
+    InteropObj.channelDataSubstore = []
+    InteropObj.chainValidatorsSubstore = []
+    ownChainAccount = {
+        "name": CHAIN_NAME_MAINCHAIN,
+        "chainId": getMainchainID(),
+        "nonce": 0
+    }
+    InteropObj.ownChainDataSubstore = [{
+        "storeKey": EMPTY_BYTES,
+        "storeValue": ownChainAccount}]
```

Comment:
I think some serialization step is missing here, i.e.,
`encode(ownChainAccountSchema, ownChainAccount)`